### PR TITLE
refactor: update cmdlets

### PR DESCRIPTION
**Pull request by bhumitra**
In order to have a good experience with our community, we recommend that you read the [contributing guidelines](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-certificate-management/blob/main/CONTRIBUTING_DCO.md) for making a pull request.

**Summary of Pull Request**

- Updated code to handle non-vSAN environments when putting the host in maintenance mode. 
- Updated function names to `Set-EsxiCertificateMode` and `Get-EsxiCertificateMode`.
- Removed `NotResponding` state from `Set-EsxiConnectionState`.
- Updated documentation.

**Type of Pull Request**

- [x] This is a bug fix.
- [x] This is an enhancement or feature.
- [ ] This is a code style / formatting update.
- [ ] This is a documentation update.
- [x] This is a refactoring update.
- [ ] This is a chore update
- [ ] This is something else.
      Please describe:

**Related to Existing Issues**

Issue Number: N/A

**Test and Documentation Coverage**

- [x] Tests have been completed (for bug fixes / features).
- [x] Documentation has been added / updated (for bug fixes / features).

**Breaking Changes?**

- [ ] Yes, there are breaking changes.
- [x] No, there are no breaking changes.

### CODE REVIEW

File: docs/functions/Get-EsxiCertificateMode.md [hunk 1]
```diff
@@ -1,104 +1,104 @@
...
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
-
-## INPUTS
-
-## OUTPUTS
-
-## NOTES
-
-## RELATED LINKS
+# Get-vCenterCertManagementMode
```
Comment (tenthirtyam) [inline docs/functions/Get-EsxiCertificateMode.md:1]: Need to update function names.

**Review by tenthirtyam — CHANGES_REQUESTED** [review]
Need to update function names in `Get-EsxiCertificateMode.md`.

File: docs/functions/Get-EsxiCertificateMode.md (hunk 1 above)
Comment (bhumitra) [reply docs/functions/Get-EsxiCertificateMode.md]: done.

**Review by tenthirtyam — APPROVED** [review]
LGTM! Thanks for adding the vSAN check and updating the cmdlet names.

**Pull request merged**